### ISSUE TITLE
build fails with current dmd beta (2.069-b1)

### ISSUE THREAD

**Issue by Extrawurst** [rejectedsoftware/vibe.d#1291]
```
vibe-d 0.7.25: building configuration "libevent"...
../../../.dub/packages/vibe-d-0.7.25/source/vibe/core/log.d(262,64): Deprecation: function std.datetime.SysTime.fracSec is deprecated - Please use fracSecs (with an s) rather than fracSec (without an s). It returns a Duration instead of a FracSec, as FracSec is being deprecated.
../../../.dub/packages/vibe-d-0.7.25/source/vibe/core/log.d(530,37): Deprecation: function std.datetime.SysTime.fracSec is deprecated - Please use fracSecs (with an s) rather than fracSec (without an s). It returns a Duration instead of a FracSec, as FracSec is being deprecated.
../../../.dub/packages/vibe-d-0.7.25/source/vibe/core/log.d(530,3): Deprecation: function std.datetime.SysTime.fracSec is deprecated - Please use fracSecs (with an s) rather than fracSec (without an s). It takes a Duration instead of a FracSec, as FracSec is being deprecated.
../../../.dub/packages/vibe-d-0.7.25/source/vibe/core/log.d(571,21): Deprecation: constructor std.datetime.SysTime.this is deprecated - Please use the overload which takes a Duration instead of a FracSec.
../../../.dub/packages/vibe-d-0.7.25/source/vibe/internal/meta/typetuple.d(113,4): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/templ/diet.d(324,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1285,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1302,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1303,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1320,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1351,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1302,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1303,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1320,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1351,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1302,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1303,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1320,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1351,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1302,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1303,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1320,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1351,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1320,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1351,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1320,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1351,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1320,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1351,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1320,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1351,3): Warning: statement is not reachable
../../../.dub/packages/vibe-d-0.7.25/source/vibe/web/rest.d(1364,2): Warning: statement is not reachable
```

**Comment by s-ludwig** [comment]
Already solved on master (f2abbd293dbbe5a07182902c2207f68d0fda580f, 84eaadf74348157ca52113a5c7138bc02d090717, 3bd96e36c74b7b468c5a19cc0cdb53223f093473), I'll tag a new alpha release shortly.

**Comment by Geod24** [comment]
Can be closed I guess ?
